easybuild/easyconfigs/b/BEDTools/BEDTools-2.30.0-GCC-11.2.0.eb

```
@@ -0,0 +1,43 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.30.0'
+
+homepage = "https://bedtools.readthedocs.io/"
+description = """BEDTools: a powerful toolset for genome arithmetic.
+The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
+computing coverage.
+The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/arq5x/bedtools2/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['333ad1ffcdc6e36005b4d6c9290677986ee97871cff92ed821c1b643d38150b8']
+
+builddependencies = [('Python', '3.9.6')]
+
+dependencies = [
+    ('XZ', '5.2.5'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.8'),
+    ('BamTools', '2.5.2'),
+]
+
+buildopts = 'CXX="$CXX"'
+
+files_to_copy = ["bin", "docs", "data", "genomes", "scripts", "test"]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['bedtools', 'pairToBed', 'mergeBed', 'bedToBam', 'fastaFromBed']],
+    'dirs': files_to_copy,
+}
+
```

@sassy-crick can we add a simple `sanity_check_command`, e.g. `--help` or `--version`?
```suggestion

sanity_check_commands = ['bedtools --help']

```

---

Good idea! I should do that from now on. That raises the question if we also should add that to the already merged PR [14344](https://github.com/easybuilders/easybuild-easyconfigs/pull/14344) ?

---

Ah, forgot to ask that for that one, indeed. Would have been useful, but we could do that in a separate/new PR, or for the next version of BamTools.